core/src/components/divider/divider.common.scss

```
@@ -0,0 +1,41 @@
+@import "../../themes/native/native.globals";
```

Since this is the common file then we import the exact path that is needed. Especially since `native.globals` is for the native (iOS and md) only. It shouldn't be used with ionic.

```suggestion
@import "../../themes/mixins";
```